### ISSUE TITLE
Register on packagist

### ISSUE THREAD

**Issue by barryvdh** [magento-hackathon/magento2-toolbar#9]
So should this be registered on Packagist already? Should make it easier to test/contribute perhaps? And it's going to be on there anyways, right? Or is there a alternative package listing we need to use for M2?

**Comment by elzekool** [comment]
With the default DataCollectors extracted we have a 0.9 version I guess. I'll try to find some spare time this weekend to add some unit tests and clean up the code a bit / add documentation where still needed. Then it's a good time to add it to packagist. Magento 2 has his own package repo if you are distributing it trough them, but I like packagist for distributing. Maybe @jissereitsma could say something usefull about this?

**Comment by barryvdh** [comment]
0.9? Shouldn't we just start with 0.1.0 ;)
Also still need to think about config, but 0.x releases at least make it easier to test integrations.

**Comment by elzekool** [comment]
The actual number is not so important for me. I think 0.9.x is a good version number idicating that something is not stable but not as green that you should keep away from it. To my knowledge this is allowed under the simver rules as everything below 1.0.0 is form free, but don't pin me on it, I could be wrong, and I'm okay with every version we should assign to it (as long it's below 1.0 for now).

**Comment by barryvdh** [comment]
Yeah bit then when we create a change to the API and we need a bump, we'll get 0.10, 0.11 etc, so don't really see why we should not just do 0.1 and go to 1.0-dev when nearing stable.

**Comment by barryvdh** [comment]
So can I assume there is not interest to maintain this? Then I'll deprecate this and create a fork in my own organisation.
